packages/block-editor/src/components/unit-control/stories/index.story.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import UnitControl from '../';
+
+const meta = {
+	title: 'BlockEditor/UnitControl',
+	component: UnitControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'UnitControl allows the user to set a numeric quantity as well as a unit.',
+			},
+		},
+	},
+	argTypes: {
+		onChange: {
+			action: 'onChange',
+			description: 'Callback function when the value changes.',
+			table: {
+				type: { summary: 'function' },
+			},
+		},
+		onUnitChange: {
+			action: 'onUnitChange',
+			description: 'Callback function when the unit changes.',
+			table: {
+				type: { summary: 'function' },
+			},
+		},
+		labelPosition: {
+			control: 'radio',
+			options: [ 'top', 'side', 'bottom', 'edge' ],
+			description: 'The position of the label.',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		label: {
+			control: 'text',
+			description: 'The label for the control.',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		value: {
+			control: { type: null },
+			description: 'The value of the control.',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		size: {
+			control: 'radio',
+			options: [ 'default', 'small' ],
+			description: 'The size of the control.',
+			table: {
+				type: { summary: 'string' },
+				defaultValue: { summary: 'default' },
+			},
+		},
+		disabled: {
+			control: 'boolean',
+			description: 'Whether the control is disabled.',
+			table: {
+				type: { summary: 'boolean' },
+				defaultValue: { summary: false },
+			},
+		},
+		disabledUnits: {
```

Comment:
```suggestion
		disableUnits: {
```

Sorry, I made a mistake with the prop name 😅 It would be nice if we could fix [this](https://github.com/WordPress/gutenberg/blob/3416bf4b0db6679b86e8e4226cbdb0d3387b25d7/packages/block-editor/src/components/unit-control/README.md#L37) together too.